src/server/graphql/types/UserMemoType.js

```diff
@@ -0,0 +1,12 @@
+import {GraphQLEnumType} from 'graphql';
+import {FACILITATOR_REQUEST} from 'universal/subscriptions/constants';
+
+const UserMemoType = new GraphQLEnumType({
+  name: 'UserMemoType',
```

Comment:
You gots ideas for other memos?

Reply:
yeah, I may change the name, but i got BIG plans for this.
basically, we're gonna have each user listening to a single USER_MEMO.userId channel.
Then, we can pass in anything we want to them, notifications, toasts, new auth tokens, etc.
i'm still weighing out the pros/cons of doing this for PRESENCE. at the expense of 1 DB hit per login, we could reduce the # of presence messages we send to the whole team.